Fix typos in the "Prerequisites for Contributing" page

phildremi opened the pull request:
Separate PR since it requires renaming the entire page 😱

phildremi committed:
Fix a typo on the contribution prerequisites page

phildremi committed:
Rename the prerequisites page (to fix another typo)

RaananW reviewed (commented):
Looks great! A redirect is missing, sitemap needs to be reverted.

Thanks a lot!

phildremi committed:
Add a redirect for the renamed prerequisites page

Some users may have bookmarked or otherwise saved the old link, which would now be broken.
